### ISSUE TITLE
Should not silently destroy symbol links

### ISSUE THREAD

**Issue by huyz** [lra/mackup#1638]
This is bad.  I run this for the first time and many of my carefully crafted symbolic links are gleefully destroyed by `mackup backup`.  Symlink organization is not something the app should just silently destroy.

**Comment by jaekyeom** [comment]
Same here. "Backing up" shouldn't modify source files.

**Comment by joshmedeski** [comment]
Do you have a suggestion? This project does its best to explain what happens clearly and adds confirmation steps when overwriting existing files.

**Comment by jaekyeom** [comment]
Thanks for commenting on an old issue. I'm a bit surprised.

> adds confirmation steps when overwriting existing files

I'm not sure if I remember it incorrectly, but does it confirm with the user before replacing the user's dotfiles and preference files with symbolic links to the files in the Mackup directory?

Nevertheless, when I see the name "Mackup", it's command "backup", and the description about its "backup" features,
what I imagine is that the user's original files will be untouched (i.e., zero modifications), because that's what backup is about.
To be honest, I believe there could be a better name for this feature. Maybe "sync" or "link"?

**Comment by joshmedeski** [comment]
The tagline for this project is

> Keep your application settings in sync

@lra has done a great job explaining how it works in the [README](https://github.com/lra/mackup#bullsht-what-does-it-really-do-to-my-files).

Dotfiles are weird and hard to back up. There are other solutions out there, but I've found mackup to be the easiest to understand and simplest to set up. Hope that helps.

**Comment by joshmedeski** [comment]
I've started a discussion on adding a `sync` command, we'll see what the community thinks. https://github.com/lra/mackup/discussions/1834

**Comment by jaekyeom** [comment]
> The tagline for this project is
> 
> > Keep your application settings in sync
> 
> @lra has done a great job explaining how it works in the [README](https://github.com/lra/mackup#bullsht-what-does-it-really-do-to-my-files).

Yeah, it was my fault I didn't read that section carefully, but when I saw the "backup" command, I assumed "keeping my application settings in sync" would be done with copying + cloud storage (or git). Or maybe because I wanted that feature.

Thanks for your kind answers and opening the discussion.
